### DISCUSSION TITLE
Github pages - remove default custom domain

### DISCUSSION THREAD

**Discussion by ogigg** [community/community#22046] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/c/fc79e5bf77e0fac6dd8e237b4e7a07ecf358378d.png" data-download-href="/uploads/short-url/A1vuaEggdahELIHiceJW1qUfk97.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/37368740/181068457-fceaf93a-515c-43d0-bfd9-591b2e91194f.png" alt="image" data-base62-sha1="A1vuaEggdahELIHiceJW1qUfk97" width="690" height="454" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc79e5bf77e0fac6dd8e237b4e7a07ecf358378d_2_690x454.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc79e5bf77e0fac6dd8e237b4e7a07ecf358378d_2_1035x681.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc79e5bf77e0fac6dd8e237b4e7a07ecf358378d_2_1380x908.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc79e5bf77e0fac6dd8e237b4e7a07ecf358378d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1590×1048 99.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
Hello, I have a problem with github pages. How to remove custom domain? It looks like it’s predefined, for some reason It’s <a href="http://ogig.me" rel="noopener nofollow ugc">ogig.me</a> instead of <a href="http://ogigg.github.io" rel="noopener nofollow ugc">ogigg.github.io</a> and I can’t change it. I tried setting it to random address and deleting it but it still going back to <a href="http://ogig.me" rel="noopener nofollow ugc">ogig.me</a> instead of <a href="http://ogigg.github.io" rel="noopener nofollow ugc">ogigg.github.io</a>. I have not found any info in the support section. I do not own this domain anymore and I want <a href="http://github.io" rel="noopener nofollow ugc">http://github.io</a> back :frowning:
<p>Can you help me? Repo url: <a href="https://github.com/ogigg/teamdeck-helper/tree/gh-pages" class="inline-onebox" rel="noopener nofollow ugc">GitHub - ogigg/teamdeck-helper at gh-pages</a></p>

**Comment by yoannchaudet** [comment]
<p>Hello!</p>
<p>Pages has the concept of user sites and project sites.</p>
<p>Your user sites lives in <a href="https://github.com/ogigg/ogigg.github.io" class="inline-onebox">GitHub - ogigg/ogigg.github.io</a> and if you set a custom domain on it (like it is currently the case), it will apply to all your project sites as well.</p>

**Reply by haorenfsa** [reply]
That's a bit unreasonable. I also run into this problem. But now our organization have hundreds of repos, how cloud I find out which one is using the custom url?  Would I have to check all repos one by one ?
